platform-ajax/src/main/typescript/com/softicar/platform/ajax/engine/DragAndDrop.ts

```diff
@@ -1,46 +1,50 @@
 
-function makeDraggable(draggedNode: HTMLElement, initNode: HTMLElement, notifyOnDrop: boolean) {
-	new DragContext(draggedNode, notifyOnDrop).setup(initNode);
+function makeDraggable(movedNode: HTMLElement, draggedNode: HTMLElement, limitingNode: HTMLElement | null, notifyOnDrop: boolean) {
```

Comment:
I find the renaming of the `draggedNode` and `initNode` confusing, maybe `draggedNode` and `dragInitiationNode`?

Reply:
To me, the "initiation" term seems quite puzzling. It does not seem to imply that the user interacts with this thing (which _should_ be implied, IMO... otherwise, we need a lengthy Javadoc explanation).
That being said, I think that calling the first node "dragged node" is a misnomer: The user does not (directly) perform a drag-and-drop operation on this node. It is merely _moved_ because they perform a drag-and-drop operation on a _different_ node.
Please reconsider.

Reply:
Renamed stuff as discussed.